protocol/x/clob/memclob/memclob.go

```diff
@@ -1890,9 +1889,9 @@ func updateResultToOrderStatus(updateResult satypes.UpdateResult) types.OrderSta
 	return types.Undercollateralized
 }
 
-// getOrderRemainingAmount returns the remaining amount of an order (its size minus its filled amount).
+// GetOrderRemainingAmount returns the remaining amount of an order (its size minus its filled amount).
 // It also returns a boolean indicating whether the remaining amount is positive (true) or not (false).
-func (m *MemClobPriceTimePriority) getOrderRemainingAmount(
+func (m *MemClobPriceTimePriority) GetOrderRemainingAmount(
```

Comment:
Technically, this function doesn't use any memclob functions/datastructures, it's on the clobKeeper. Maybe we should move it out in a future PR.

Reply:
nvm, I see the other comment.